swift/ios-shoppe-demo/ServiceManagers/APIService.swift

```diff
@@ -7,18 +7,18 @@
 //
 
 import Foundation
+import UIKit
 
 class APIService {
     static var sharedInstance = APIService()
 
     var baseURL: String {
-        guard let webserviceURL = getPlist(withName: "Preferences") else {
-            print("No web services URL found")
-            return ""
-        }
-        return webserviceURL[0]
+        return getPreferencesArray()[0]
```

Comment:
Curious if there's a way to use key/value pairs with the Plist rather than an array. It's possible that the order of values in the Plist could change, which means that you'll be fetching the wrong value in your code.

Reply:
Your 100% I will switch it